frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -323,8 +345,16 @@ const getActiveFilters = (keywordFilter, groupFilters, activeFilters) => {
   return activeFilters;
 };
 
+export const updateActiveDisplayType = (activeDisplayType, displayType, value) => {
```

Comment:
Add types for the parameters

Reply:
It's a jsx file

Reply:
Oh missed that. Ignore this comment then :P